### PR TITLE
Update SegmentedControlIOS.ios.js

### PR DESCRIPTION
**Pull request by pedramsaleh**
Updating the comments to clarify that the selectedIndex prop is not just for pre-selecting an index, but that it can also be used to programmatically change the value of the selected index.

### CODE REVIEW

**Comment by janicduplessis** [comment]
Hey @pedramsaleh, thanks for submitting this!

What about we simplify the prop description a bit and add a usage example at the top with a simplified version of what I linked on Stack Overflow?

For the prop desc you could just remove the 'pre-selected' wording that is confusing and just say the 'selected index'.

For the example you could add the note about changing the selectedIndex programmatically after the code example.

Code example could look like this:
```
<SegmentedControlIOS
  values={['One', 'Two']}
  selectedIndex={this.state.selectedIndex}
  onChange={(event) => {
    this.setState({selectedIndex: event.nativeEvent.selectedSegmentIndex});
  }}
/>
```
(Note about using `setState` to change the selected value programically)

You can add it in the comment at the top of the component and use markdown for header/code blocks
(see https://github.com/facebook/react-native/blob/master/Libraries/Components/StatusBar/StatusBar.js#L75 as an example of this)

**Comment by pedramsaleh** [comment]
OK, I simplified the description of selectedIndex and added the example at the top, with an explanation beforehand.

**Comment by pedramsaleh** [comment]
I set the description of selectedIndex to "(pre)selected" to indicate that preselecting is an option, as well as programmatically updating on the fly.

File: Libraries/Components/SegmentedControlIOS/SegmentedControlIOS.ios.js [hunk 1]
```diff
@@ -31,6 +31,23 @@ type Event = Object;
 
 /**
  * Use `SegmentedControlIOS` to render a UISegmentedControl iOS.
+ *
+ * #### Programmatically changing selected index
+ *
+ * The selected index can be changed by the fly by assigning the
```
Comment (janicduplessis) [inline Libraries/Components/SegmentedControlIOS/SegmentedControlIOS.ios.js]: Typo: Should be 'on the fly' instead of 'by the fly'

**Comment by janicduplessis** [comment]
Thanks! This is great, just fix the little typo and this is good to go :)

**Comment by pedramsaleh** [comment]
Haha!  Can't believe I didn't catch that.  OK, fixed.

**Comment by mkonicek** [comment]
Thanks for improving the docs @pedramsaleh and thanks for the review @janicduplessis!

@facebook-github-bot shipit

**Pull request closed without merging**